inductiva/fluids/scenarios/wind_terrain/terrain.py

```diff
@@ -0,0 +1,147 @@
+"""Represent a complex terrain profile."""
```

Comment:
I think this should not live under fluid/scenarios.
I think we need to create another package just for this sort of classes.

inductiva/world

Let's open an issue on this, but for now please move it out of fluids and place it in world (or another package you see more fit.)

Reply:
Seems appropriate :)